packages/ord-connect/src/lib/signMessage.ts

```diff
@@ -0,0 +1,40 @@
+import { ordit } from "@sadoprotocol/ordit-sdk";
+import { Network, Wallet } from "../providers/OrdContext.tsx";
+
+interface SignMessageParams {
+  message: string;
+  wallet: Wallet;
+  address: string | Record<string, string>;
+  network: Network;
+}
+
+export default async function getSignedMessage({
```

Comment:
i think the function name should be the same as file name which is `signMessage`